tools/XCConfigDumper/Package.swift

```diff
@@ -0,0 +1,35 @@
+// swift-tools-version:5.1
```

Comment:
Any reason to add an additional build system and tooling? I'm not sure how this will end up working end to end ( e.g. actually integrating to XCHammer and building projects )